service/src/main/java/bio/terra/landingzone/library/landingzones/definition/factories/CromwellBaseResourcesFactory.java

```diff
@@ -186,18 +188,14 @@ public Deployable definition(DefinitionContext definitionContext) {
                   nameGenerator.nextName(ResourceNameGenerator.MAX_POSTGRESQL_SERVER_NAME_LENGTH))
               .withRegion(resourceGroup.region())
               .withExistingResourceGroup(resourceGroup.name())
-              .withProperties(
-                  new ServerPropertiesForDefaultCreate()
-                      .withAdministratorLogin(
-                          parametersResolver.getValue(ParametersNames.POSTGRES_DB_ADMIN.name()))
-                      .withAdministratorLoginPassword(
-                          parametersResolver.getValue(ParametersNames.POSTGRES_DB_PASSWORD.name()))
-                      .withVersion(ServerVersion.ONE_ONE)
-                      .withPublicNetworkAccess(PublicNetworkAccessEnum.DISABLED))
-              .withSku(
-                  new Sku()
-                      .withName(
-                          parametersResolver.getValue(ParametersNames.POSTGRES_SERVER_SKU.name())));
+              .withVersion(ServerVersion.ONE_FOUR)
+              .withAdministratorLogin(
+                  parametersResolver.getValue(
+                      CromwellBaseResourcesFactory.ParametersNames.POSTGRES_DB_ADMIN.name()))
+              .withAdministratorLoginPassword(
+                  parametersResolver.getValue(
+                      CromwellBaseResourcesFactory.ParametersNames.POSTGRES_DB_PASSWORD.name()))
+              .withSku(new Sku().withName("GP_Gen5_2"));
```

Comment:
should this use `ParametersNames.POSTGRES_SERVER_SKU.name()`?

Reply:
don't worry about this, it is an unused code path that will be removed shortly